docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #62.            if direction == 'right': #the name 'right' has conflict with the inequality <code>cands_idx &lt; max(0, i - excl_zone)</code>! idea?
</pre><p>What is the conflict? Can you please explain?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I think my wording was not correct. I was talking about the readability of the code. Since we are saying the word `right` and then do `cands_idx < max(0, i - excl_zone)`, which gives all candidates to the left of subsequence `i` (I know `i` is the right neighbor of those candidates). I was wondering if it is okay as is.

Reply:
<p>I am okay with it I think</p>